ipywidgets/static/widgets/js/manager.js

```diff
@@ -280,11 +280,26 @@ define([
         /**
          * Handle when a comm is opened.
          */
-        return this.create_model({
-            model_name: msg.content.data.model_name,
-            model_module: msg.content.data.model_module,
-            comm: comm,
-        }).catch(utils.reject("Couldn't create a model.", true));
+        // For < 4.x
+        var model;
+        if (msg.content.data.model_name) {
```

Comment:
I think we broke it in a later commit, sorry @shashi !  @SylvainCorlay and I will try to be better about things like this.  It was removed in https://github.com/ipython/ipywidgets/commit/474f08cbcdc47a573297447af8474ec0333c9687